tests/components/html5/test_notify.py

```diff
@@ -83,166 +83,174 @@ async def mock_client(hass, hass_client, registrations=None):
     return await hass_client()
 
 
-class TestHtml5Notify:
```

Comment:
These tests could use a lot more refactoring.  I did enough to get them to pass but there is more cleanup that should be done in a future PR